### PR TITLE
Add SignIn frontend

### PR DESCRIPTION
**Pull request by VortexExpansion**
Functionality to be added soon

### CODE REVIEW

**Comment by VortexExpansion** [comment]
> Functionality to be added soon

Done.

**Comment by VortexExpansion** [comment]
> Functionality to be added soon

Done.

File: app/src/main/res/layout/activity_sign_in.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            android:layout_width="200dp"
+            android:layout_height="200dp"
+            app:srcCompat="@drawable/varchas_dummy_logo"
+            android:contentDescription="@string/todo" />
+
+        <Button
+            android:id="@+id/btnSignIn"
+            android:layout_width="150dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="200dp"
```
Comment (imraghav20) [inline app/src/main/res/layout/activity_sign_in.xml]: Too much margin looks ugly.
Reduce to 100dp

Comment (VortexExpansion) [reply app/src/main/res/layout/activity_sign_in.xml]: Done

**Pull request merged**